content/en/docs/concepts/overview/working-with-objects/annotations.md

```diff
@@ -53,11 +53,22 @@ Here are some examples of information that could be recorded in annotations:
 * Phone or pager numbers of persons responsible, or directory entries that
   specify where that information can be found, such as a team web site.
 
+* Directives from the end-user to the implementations to modify behavior or
+  engage non-standard features.
+
 Instead of using annotations, you could store this type of information in an
 external database or directory, but that would make it much harder to produce
 shared client libraries and tools for deployment, management, introspection,
 and the like.
 
+## Syntax and character set
+
+_Annotations_ are key/value pairs. Valid annotation keys have two segments: an optional prefix and name, separated by a slash (`/`).  The name segment is required and must be 63 characters or less, beginning and ending with an alphanumeric character (`[a-z0-9A-Z]`) with dashes (`-`), underscores (`_`), dots (`.`), and alphanumerics between.  The prefix is optional.  If specified, the prefix must be a DNS subdomain: a series of DNS labels separated by dots (`.`), not longer than 253 characters in total, followed by a slash (`/`).
+
```

Comment:
In general, looks good.
super nit: There are a few places where there are two spaces after the sentence period instead of one space. The same for the Labels description below.

Reply:
fixed in repush